stable/cluster-autoscaler/values.yaml

```diff
@@ -40,9 +40,9 @@ image:
 tolerations: []
 
 extraArgs:
-  v: 4
-  stderrthreshold: info
-  logtostderr: true
+  # v: 4
```

Comment:
I don't see the point of adding this - it was part of https://github.com/helm/charts/pull/4614 - but this should be configurable by users, not the default chart value ...

Reply:
Let's keep this change out of this PR. Then we can merge the `dnsPolicy` related changes.

Reply:
@linki - Reverted - but why is the verbosity so high by default and could you please instruct me how to remove these extraArgs if they are set in the Chart defaults?

Reply:
@so0k I don't know why they are set to these values. I'd prefer to have it similar to other charts but I would need to check a couple. We can tackle this in a seprate PR.

In the meantime, shouldn't you be able to override the values with something that logs less, e.g. in `values.yaml`
```yaml
extraArgs:
  v: 1
```
or `--values extraArgs.v=1` etc. on the command line?

Reply:
Ok thanks!